packages/daf-did-jwt/src/message-handler.ts

```diff
@@ -19,7 +19,7 @@ export class JwtMessageHandler extends AbstractMessageHandler {
         const resolver = { resolve: (didUrl: string) => context.agent.resolveDid({ didUrl }) }
         const verified = await verifyJWT(message.raw, { resolver, audience })
         debug('Message.raw is a valid JWT')
-        message.addMetaData({ type: decoded.header.typ, value: decoded.header.alg })
+        message.addMetaData({ type: decoded.header.typ || 'JWT', value: decoded.header.alg })
```

Comment:
Do we need to make any changes here? https://github.com/uport-project/daf/blob/beta/packages/daf-w3c/src/message-handler.ts#L51

Reply:
@simonas-notcat Yes, definitely, but it's not in the scope of this PR

daf-w3c benefits from this PR, though